src/server/store/ActivityStore.ts

```diff
@@ -32,29 +57,34 @@ export class ActivityStore {
     await this.db.del(key)
   }
 
-  get indexesDB (): AbstractLevel<any, string, any> {
-    return this.db
-      .sublevel('indexes', { valueEncoding: 'json' })
-  }
-
-  get publishedIndex (): AbstractLevel<any, string, any> {
-    return this.indexesDB
-      .sublevel('published', { valueEncoding: 'json' })
-  }
-
-  makeIndexKey (activity: APActivity): string {
-    const publishedAt = getPublished(activity.published)
-    if (typeof activity.id !== 'string') throw createError(400, 'Missing ID in Actvity')
-    return `${publishedAt.toISOString()}-${activity.id}`
-  }
-
   async addToIndex (activity: APActivity): Promise<void> {
-    await this.publishedIndex.put(this.makeIndexKey(activity), activity.id)
+    const { published, id } = activity
+    if (published === undefined || id === undefined) return
```

Comment:
does this silently ignore activities without a published attribute? are they considered invalid? (i see the `add()` function sets a default date if it missing)

Reply:
It's mostly to make typescript happy we can safely ignore it